enterprise/frontend/src/embedding-sdk-bundle/components/public/SdkQuestion/SdkQuestion.unit.spec.tsx

```diff
@@ -325,8 +325,7 @@ describe("InteractiveQuestion", () => {
 
     const columnTitle = screen.getByTestId("column_title");
     await userEvent.clear(columnTitle);
-    await userEvent.type(columnTitle, "A New Test Column");
-    await userEvent.tab();
+    await userEvent.paste("A New Test Column");
```

Comment:
Speed up the test a bit doesn't hurt. As pasting is generally faster than typing.